### ISSUE TITLE
orgmode exports for pdflatex

### ISSUE THREAD

**Issue by dfc** [crypto101/book#204]
_I apologize that I do not have the solution for this one. I have an idea about what needs to happen but my lisp/orgmode is weak._

Orgmode has a default-packages-alist that it uses for exporting. The problem is that this list is set up with pdflatex in mind so it automatically inserts fontenc and inputenc in the preamble. fontenc/inputenc are relics of the era when UTF was scary and new. lua/xe-latex have no problem with unicode. I have a hunch that this might have been a problem with the pkcs references.  lualatex complains about this in the logfile:

```
*** you should *not* be loading the inputenc package
*** XeTeX expects the source to be in UTF8 encoding
*** some features of other encodings may conflict, resulting in poor output.

```

the default packages are listed in `$ORG_DIR/org.el`

```

(defcustom org-latex-default-packages-alist
  '(("AUTO" "inputenc"  t)
    ("T1"   "fontenc"   t)
    (""     "fixltx2e"  nil)
    (""     "graphicx"  t)
    (""     "longtable" nil)
    (""     "float"     nil)
    (""     "wrapfig"   nil)
    (""     "rotating"  nil)
    ("normalem" "ulem"  t)
    (""     "amsmath"   t)
    (""     "textcomp"  t)
    (""     "marvosym"  t)
    (""     "wasysym"   t)
    (""     "amssymb"   t)
    (""     "hyperref"  nil)
    "\\tolerance=1000")

```

The other problem child is listed last, that tolerance line is basically a license for ugliness. There was one commit a long time ago where you lamented the layout of a figure and commented that latex was not supposed to do such ugly things. I think that tolerance line is the problem. I made two pdfs with and without the tolerance line. 13 of 252 pages were different. I made a imgur album with the diffs: http://imgur.com/a/C5tMv

Full disclosure/gotchas first: The only place the default tolerance is bad is on the license page it is the first page in the album. lualatex cant deal with the length of Attribution-Noncommercial. Everything else is microsoft word layout quality problems until you get to the third and second to last page. With the tolerance=1000 lualatex thinks it is okay to hyphenate "equation" and have the second half of the word start on the begining of the next page. This should never happen. I am not really sure why orgmode has this value tweaked so high by default. 

I am sitting on one more pull request to fix the math font so that it resembles the main body font and is much more legible. Once the fontenc/inputenc are not loaded by default I will submit the math font fix. It gets rid of the disparity between the body text and the math text, it is a lot less jarring.  For some strange reason lualatex on OSX ccomplains one out of ten times when I have the good math font loaded with fontenc/inputenc. Texlive on Debian--is obviously awesomer-- so it does not have the occasional mood swings;)

I am sure you know how to use google, but to give you a jumpstart here is someone discussing the problem and a solution: https://lists.gnu.org/archive/html/emacs-orgmode/2012-11/msg00222.html

tl;dr: fontenc/inputenc need to go back to the 90s and not be included in the preamble. The tolerance thing is a "nice to have done correctly" but not as much of a problem as the fontenc/inputenc. 

As always, thanks for creating and giving us such an awesome book.
